src/app/components/InfoPage/style.scss

```diff
@@ -0,0 +1,5 @@
+:local {
+  .container {
+    box-sizing: border-box;
```

Comment:
We all need this style for every element on every page, so my suggestion is deleting this file and move `box-sizing: border-box;` to `src/app/style.scss`

![image](https://user-images.githubusercontent.com/43344276/94687683-2b68d880-035f-11eb-9ad1-38a64c167f2f.png)